lib/productive.rb

```diff
@@ -72,10 +70,9 @@
 require 'productive/resources/reports/service_report'
 require 'productive/resources/reports/task_report'
 require 'productive/resources/reports/time_entry_report'
+require 'productive/resources/reports/time_report'
```

Comment:
@BojanOstic Ehh ... znas sta sam sad uocio -> ima gore `new_time_report` resource: a to je taj. Ista stvar sa ovim `new_salary_report`. Trebali bi polako deprecatat taj stari naming. Znaci dodaj gore jos `/reports/salary_report` resource -> i onda po exportu i maileru treba prokopati da krenemo koristiti ovaj bolji resource naming bez `New` prefixa.